### PR TITLE
fix: 🐛 [JIRA: HCPSDKFIORIUIKIT-2478] banner message style

### PR DESCRIPTION
**Pull request by xiaoyu0722**
1. Add icon support for banner message.
2. Add new cases for long title and different styles.

### CODE REVIEW

File: Apps/Examples/Examples/FioriSwiftUICore/BannerMessage/BannerMessageModifierExample.swift [hunk 1]
```diff
@@ -59,6 +70,123 @@ struct BannerMessageModifierExample: View {
...
+            return BannerMessageInformationStyle()
+        default:
+            return BannerMessageNeutralStyle()
+        }
+    }
+}
+
+// MARK: - Negative Style
+
+struct BannerMessageNegativeStyle: BannerMessageStyle {
```
Comment (billzhou0223) [inline Apps/Examples/Examples/FioriSwiftUICore/BannerMessage/BannerMessageModifierExample.swift:96]: Are these styles required by design spec? Should we add these styles into SDK to make them reusable?

**Comment by janhuachu** [comment]
@xiaoyu0722  HCPSDKFIORIUIKIT-1940 is for ProfileHeader. The changes do not seem to have anything to do with ProfileHeader.
Could you check your JIRA item id.

**Comment by xiaoyu0722** [comment]
> @xiaoyu0722 HCPSDKFIORIUIKIT-1940 is for ProfileHeader. The changes do not seem to have anything to do with ProfileHeader. Could you check your JIRA item id.

Sorry, my mistake, I have updated the commit message.

File: Apps/Examples/Examples/FioriSwiftUICore/BannerMessage/BannerMessageModifierExample.swift (hunk 1 above)
Comment (xiaoyu0722) [reply Apps/Examples/Examples/FioriSwiftUICore/BannerMessage/BannerMessageModifierExample.swift:96]: Emm, actually I'm not sure if these are needed, that's why I put them in example but not sdk.
Basic no useful information in story [BannerMessage](https://jira.tools.sap/browse/HCPSDKFIORIUIKIT-2478).
I will double check with Grace that where can I find the finalized design for `BannerMessage`.

Comment (xiaoyu0722) [reply Apps/Examples/Examples/FioriSwiftUICore/BannerMessage/BannerMessageModifierExample.swift]: These styles are for next release, have been removed now.

**Review by billzhou0223 — APPROVED** [review]

**Pull request merged**